cli/src/lib.rs

```diff
@@ -104,6 +104,7 @@ where
         query_judge: QueryJudgeBoxed,
     ) -> Result<Self> {
         let broker = Broker::new();
+        // TODO: make entrypoint use proxy
```

Comment:
Since this is not a draft PR, I'd be much more comfortable with an issue.

Reply:
Created #2614.